stellar-multitenant/pkg/tenant/manager.go

```diff
@@ -236,6 +238,36 @@ func (m *Manager) SoftDeleteTenantByID(ctx context.Context, tenantID string) (*T
 	return &t, nil
 }
 
+// DeactivateTenantDistributionAccount sets a distribution account of status ACTIVE to PENDING_USER_ACTIVATION for the given tenant id,
+// and is only used in the case where the distribution account is of type CircleDBVault.
+func (m *Manager) DeactivateTenantDistributionAccount(ctx context.Context, tenantID string) error {
+	updateQuery := `
+		UPDATE tenants t
+		SET
+			distribution_account_status = 'PENDING_USER_ACTIVATION'
+	`
+
+	queryParams := &QueryParams{
```

Comment:
I think queryParams should be used when we have to conditionally stitch the query string together depending on optional parameters. 

In cases (like this) where the query doesn't change between calls, we should keep things explicit and just write the full query.

Reply:
I think that'd be simpler as well.